src/main/webapp/app/entities/event/event.tsx

```diff
@@ -14,6 +14,7 @@ import {
   JhiItemCount
 } from 'react-jhipster';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons/faEllipsisH';
```

Comment:
this one should be import in icon-loader.ts

Reply:
Resolved in 7f7222ecea72e3c604f27767af7c3e4f433c1c1d